scripts/flattener.py

```diff
@@ -570,6 +570,30 @@ def clean_obs(glob):
 		}
 	}, inplace=True)
 
+	# Concatenate of flowcell and lane for Tier1 metadata
+	if hcatier1:
+		if 'library_sequencing_run' in glob.cxg_obs.columns:
+			if glob.cxg_obs['library_sequencing_run'].dtype != 'category':
+				glob.cxg_obs['library_sequencing_run'] = glob.cxg_obs['library_sequencing_run'].astype('category')
+			for seq_run in glob.cxg_obs['library_sequencing_run'].cat.categories:
+				if seq_run != 'unknown':
+					runs = []
+					orig_cat = seq_run
+					seq_run = seq_run.replace("pooled ","")
+					seq_run = seq_run.replace("'", '"')
+					print(seq_run)
```

Comment:
Just need to clean up the print, then good to merge

Reply:
Is there a face palm emoji?